frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -148,6 +143,12 @@ export function FieldValuesWidgetInner({
   optionRenderer,
   layoutRenderer,
 }: IFieldValuesWidgetProps) {
+  const { ref, width: elementWidth } = useElementSize();
```

Comment:
This replaces the `ExplicitSize` call to hopefully remove the `cannot find node on unmounted component` errors